AutomatedTesting/Gem/PythonTests/Physics/tests/material/Material_LibraryUpdatedAcrossLevels.py

```diff
@@ -252,10 +252,8 @@ def modify_material_library():
     for test in test_list:
         # 1) Open the correct level for the test
         helper.open_level(
-            "physics",
-            "Material_LibraryUpdatedAcrossLevels\\Material_LibraryUpdatedAcrossLevels_{}".format(
-                test.level_index
-            ),
+            "Physics",
+            os.path.join("Material_LibraryUpdatedAcrossLevels", str(test.level_index))
```

Comment:
This join is not equivalent to the original, which was 'Material_LibraryUpdatedAcrossLevels_{}', is this correct? Shouldnt it be
```suggestion
            os.path.join("Material_LibraryUpdatedAcrossLevels", f"Material_LibraryUpdatedAcrossLevels_{test.level_index}") str(test.level_index))
```
?

Reply:
The level itself didn't match before because it had an id, you can see in the PR the level being saved with the proper name, matching the new changes.

Before:
**Script**: Material_LibraryUpdatedAcrossLevels/Material_LibraryUpdatedAcrossLevels_0    **Level**: Material_LibraryUpdatedAcrossLevels/C15425935_Material_LibraryUpdatedAcrossLevels_0

Now:
**Script**: Material_LibraryUpdatedAcrossLevels/0    **Level**: Material_LibraryUpdatedAcrossLevels/0

Now they match, it's shorter and follows the naming convention of other tests.